src/routes/store/gift-cards/[slug]/+page.svelte

```diff
@@ -30,6 +30,21 @@
     }`;
 
 	let count = 0, selectedDenomination = 0;
+
+	const getAmountWithoutDiscount = (/** @type {number} */ amount = 0, /** @type {number} */ commission_percent = 0) => { 
+
+		let amount_to_pay = amount - ((amount*commission_percent) / 100);
+
```

Comment:
Hmmmm.

Inline typing like this might seem a bit messy, especially if the params grows. It might be better to use params block like this

```
/**
 * @param {number} amount The original amount that is to be discounted 
 *  @param {number} commission_percent The integer representation of the percentage discount.
 * @returns number
 * /
const getAmountWithoutDiscount = (amount = 0, commission_percent = 0)